### DISCUSSION TITLE
Multi Azuracast Instances on One Server

### DISCUSSION THREAD

**Discussion by dekakast** [AzuraCast/AzuraCast#6920] [Q&A] [answered]
Hi, I know it is possible to host multiple sites on one server with azuracast, but is it also possible to host multiple azuracast instances on one server?

It is not clear to me at this link: https://www.azuracast.com/docs/administration/multi-site-installation/

If not possible to host multiple azuracast instances, any suggestions to change the url of an existing azuracast instance?

**Comment by dekakast** [comment]
In System Settings, can I just change the site based url, and then update my DNS records for the new url?

**Reply by unudelaradio** [reply]
Hello! If you just want to point multiple subdomains to your AzuraCast installation, you can add these in System Settings / Services / Domain Names, all separated with commas. 

And of course add A Records with the IP of your AzuraCast, in your website cPanel, for each subdomain.

**Reply by dekakast** [reply]
Thanks @unudelaradio I am not sure I follow you. If I have abc.com as my base url, then also want to use xyz.com, I do not think I can put xyz.com in the Domain Names and create a stream for xyz.com/public/radio.

**Reply by unudelaradio** [reply]
It should work. I'm using subdomains. Like subdomain.abc.com, subdomain.xyz.com etc. If you put all your subdomains in that field, no matter if they are different, it should work. Click save first, then generate SSL. This way, all your different domains will point to the same instance.

**Reply by dekakast** [reply]
But I do not see how I create a unique station stream url for subdomain.xyz.com. Every new station stream url is always tied to the base url abc.com.

**Reply by unudelaradio** [reply]
I didn't try to use Azuracast with main domains like xyz.com. What i can tell you is that when using subdomains, everything works perfect.

Every station profile created in your azuracast instance can be accessed using any subdomain address, including base URL. 

Station 1 profile will be reachable like this:

subdomain1.xyz.com/public/station_1
subdomain2.abc.com/public/station_1

Both will work. It's your choise how you share it. 

You will be able to login in your Azuracast dashboard on both addresses too. 

subdomain1.xyz.com/login
subdomain2.abc.com/login

I repeat. Don't know how this will work with main domains. My main domains are basically the station websites.

**Comment by dekakast** [comment]
I guess if I installed the original azuracast server in this directory /var/azuracast, then I could just create a new directory /var/azuracast2 and install the second azuracast server there?

**Comment by Vaalyn** [comment, marked as answer]
Just point the other domain/s to the server via an A-Record, activate the `Prefer Browser URL` setting (see screenshot) and as was described above add the domain to the let's encrypt domain names.

When you access the UI via the other domain/s the system will put the URL that you are visiting the UI from as the base URL for the links.

<img width="666" alt="image" src="https://github.com/AzuraCast/AzuraCast/assets/13745863/9862bdc6-88f2-4342-bd7b-643458bdd855">

You cannot install AzuraCast 2 times on the same server in parallel.

**Reply by t0mtaylor** [reply]
When you enable this feature (Prefer Browser URL), it is using the "relative url" (without domain) and not the "absolute url" (with base domain/url), meaning it will load regardless of the domain.

**Comment by dekakast** [comment]
Thank you @Vaalyn, but when I do all that and create the new station with the different url, the new station still defaults to the old base url

**Comment by dekakast** [comment]
@unudelaradio and @Vaalyn, I guess I spoke too soon :) while the public page in the Profile displays the old base url, the new url does work.
